src/util.h

```diff
@@ -644,7 +645,7 @@ using DeleteFnPtr = typename FunctionDeleter<T, function>::Pointer;
 std::vector<std::string> SplitString(const std::string& in, char delim);
 
 inline v8::MaybeLocal<v8::Value> ToV8Value(v8::Local<v8::Context> context,
-                                           const std::string& str,
+                                           const std::string_view& str,
```

Comment:
Why not just
```suggestion
                                           std::string_view str,
```
? Since it's just a view, ToV8Value() shouldn't be able to modify the underlying string anyways, right?

Could we also include `string_view` in here and in `src/util-inl.h` because of https://google.github.io/styleguide/cppguide.html#Include_What_You_Use?

Reply:
Sure, done :+1: